### ISSUE TITLE
Data missing

### ISSUE THREAD

**Issue by flowingrain** [agangbe/GACL#4]
In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.

**Comment by findmi** [comment]
> In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.

The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?

**Comment by flowingrain** [comment]
> > In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.
> 
> The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?

The recurrence results are as follows:
- Twitter15:
AVG_result: 0.8694|UR F1: 0.8370|NR F1: 0.9397|TR F1: 0.8516|FR F1: 0.8284
- Titter16:
AVG_result: 0.9075|UR F1: 0.8896|NR F1: 0.9118|TR F1: 0.9574|FR F1: 0.8614

**Comment by findmi** [comment]
> > > In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.
> > 
> > 
> > The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?
> 
> The recurrence results are as follows:
> 
> * Twitter15:
>   AVG_result: 0.8694|UR F1: 0.8370|NR F1: 0.9397|TR F1: 0.8516|FR F1: 0.8284
> * Titter16:
>   AVG_result: 0.9075|UR F1: 0.8896|NR F1: 0.9118|TR F1: 0.9574|FR F1: 0.8614

thank u so much

**Comment by flowingrain** [comment]
> > > > In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.
> > > 
> > > 
> > > The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?
> > 
> > 
> > The recurrence results are as follows:
> > 
> > * Twitter15:
> >   AVG_result: 0.8694|UR F1: 0.8370|NR F1: 0.9397|TR F1: 0.8516|FR F1: 0.8284
> > * Titter16:
> >   AVG_result: 0.9075|UR F1: 0.8896|NR F1: 0.9118|TR F1: 0.9574|FR F1: 0.8614
> 
> thank u so much

Randomly divide data sets instead of giving each fold an equal number of samples for each category.

> > > > In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.
> > > 
> > > 
> > > The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?
> > 
> > 
> > The recurrence results are as follows:
> > 
> > * Twitter15:
> >   AVG_result: 0.8694|UR F1: 0.8370|NR F1: 0.9397|TR F1: 0.8516|FR F1: 0.8284
> > * Titter16:
> >   AVG_result: 0.9075|UR F1: 0.8896|NR F1: 0.9118|TR F1: 0.9574|FR F1: 0.8614
> 
> thank u so much

Randomly divide data sets instead of giving each fold an equal number of samples for each category.

**Comment by flowingrain** [comment]
> > > > In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.
> > > 
> > > 
> > > The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?
> > 
> > 
> > The recurrence results are as follows:
> > 
> > * Twitter15:
> >   AVG_result: 0.8694|UR F1: 0.8370|NR F1: 0.9397|TR F1: 0.8516|FR F1: 0.8284
> > * Titter16:
> >   AVG_result: 0.9075|UR F1: 0.8896|NR F1: 0.9118|TR F1: 0.9574|FR F1: 0.8614
> 
> thank u so much

Randomly divide data sets instead of giving each fold an equal number of samples for each category.

**Comment by Dbgsaoge** [comment]
> > > In the process of reprinting the paper, I noticed that the amount of data in several datasets you provided was significantly less than that described in the article, so that the results of the model were not as good as those reported. For example, Twitter16 contains 818 source tweets, but only 661 embeds are provided. Can you provide a complete pre-trained word embedding to reproduce the model results while complying with the privacy policy? Thank you very much.
> > 
> > 
> > The author says he removed responses with less than 3 comments, and, can u tell me what is the accuracy and F1 you get?
> 
> The recurrence results are as follows:
> 
> * Twitter15:
>   AVG_result: 0.8694|UR F1: 0.8370|NR F1: 0.9397|TR F1: 0.8516|FR F1: 0.8284
> * Titter16:
>   AVG_result: 0.9075|UR F1: 0.8896|NR F1: 0.9118|TR F1: 0.9574|FR F1: 0.8614

Hi do you know what the difference among the "t15_mask_00"， "t15_mask_01", "t15_mask_02", and "t15_mask_015"？ I dont konw why the it should load data in different data_fold. And i found that the emb of the same tweet in each folder is different.
